Doppler.HtmlEditorApi/Controllers/TemplatesController.cs

```diff
@@ -15,63 +16,19 @@ namespace Doppler.HtmlEditorApi.Controllers
     [ApiController]
     public class TemplatesController : Controller
```

Comment:
Could you use POCO controllers (it is better to avoid unwanted dependencies)?

```suggestion
    public class TemplatesController
```

Reply:
Great! Fixed